docker-compose.yml

```diff
@@ -1,15 +1,23 @@
 version: '3.4'
 
-x-production: &x-production
-  RAILS_ENV: production
+x-govuk-app-env: &govuk-app
+  DISABLE_DATABASE_ENVIRONMENT_CHECK: 1
+  GDS_SSO_STRATEGY: mock
   GOVUK_APP_DOMAIN: dev.gov.uk
   GOVUK_WEBSITE_ROOT: http://www.dev.gov.uk
   GOVUK_ASSET_ROOT: http://assets-origin.dev.gov.uk
+  LOG_PATH: log/live.log
+  PLEK_SERVICE_SEARCH_URI: http://rummager.dev.gov.uk
+  RAILS_ENV: production
+  RAILS_SERVE_STATIC_FILES: "true"
   SECRET_KEY_BASE: 875c6bf4c48da9bb41f4cfd25d09bf5e2a62d88b39efc4bd9c498e6c8f61e4df740af87386f97269525e01b5f74402512eb6a4723882579f10aa95f6e2971fc2
   SECRET_TOKEN: 875c6bf4c48da9bb41f4cfd25d09bf5e2a62d88b39efc4bd9c498e6c8f61e4df740af87386f97269525e01b5f74402512eb6a4723882579f10aa95f6e2971fc2
```

Comment:
FYI I've started the process of having all apps use SECRET_KEY_BASE so we should be able to kill this soon (but not something to wait for before merging)